content/sensu-go/5.17/release-notes.md

```diff
@@ -52,6 +53,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.17.2 release notes
+
+**February 19, 2020** &mdash; The latest release of Sensu Go, version 5.17.2, is now available for download.
+This release fixes a bug that could cause enterprise HTTP routes to fail to initialize upon internal restart.
```

Comment:
I actually have no clue but do we prefer to call them _enterprise_ or _commercial_ routes now?

Reply:
You are correct -- it should be "commercial" and thank you for catching this!